### PR TITLE
Cloudinary setup and modals update

### PR DESCRIPTION
**Pull request by pmunozs**
- Cloudinary setup
- Update models
- Update views (forms and lists)
- Update ENV keys

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -4,4 +4,5 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
   has_many :creatures, dependent: :destroy
+  has_one_attached :photo
```
Comment (henderson907) [inline app/models/user.rb:7]: why is the user model requiring a photo?

File: app/views/pages/my_profile.html.erb [hunk 2]
```diff
@@ -9,7 +9,8 @@
     <% @creatures.each do |creature| %>
       <div class="col-sm d-flex justify-content-between d-flex flex-wrap">
         <div class="card co-12 col-sm-6 col-md-4 mb-3">
-            <img class="card-img-top" src="https://source.unsplash.com/500x500/?<%= creature.name %>, scary" alt="Card image cap" />
+            <%= cl_image_tag creature.photo.key, class: 'card-img-top', crop: :fill %>
+            <!-- <img class="card-img-top" src="https://source.unsplash.com/500x500/?<%= creature.name %>, scary" alt="Card image cap" /> -->
```
Comment (henderson907) [inline app/views/pages/my_profile.html.erb:13]: is this the same as the previous line but commented out? And so, is it needed?

**Review by henderson907 — CHANGES_REQUESTED** [review]
Has the schema been updated to reflect the image tags? It doesn't load automatically in git hub

File: app/models/user.rb (hunk 1 above)
Comment (pmunozs) [reply app/models/user.rb]: Removed

File: app/views/pages/my_profile.html.erb (hunk 2 above)
Comment (pmunozs) [reply app/views/pages/my_profile.html.erb]: Removed

**Pull request merged**